core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -1707,7 +1707,18 @@ public function generateRecordLabel(array $row, string $table = null, bool $prot
 					->limit(1)
 					->execute($row[$strKey]);
 
-				$args[$k] = $objRef->numRows ? $objRef->$strField : '';
+				$args[$k] = $objRef->numRows ? $objRef->$strField : '-';
```

Comment:
Why did you change the default value?

Reply:
Because this is the same default value as all the other if-conditions in the method will return. It is previously done in the `listView`, but should be unified across all views.

Reply:
This needs to be reverted, too (see below).